content/00.build/40.tooling/20.hardhat/00.introduction/02.installation.md

```diff
@@ -0,0 +1,133 @@
+---
+title: Installation
+description: How to install zksync-hardhat for your project.
+tags: ["zksync", "hardhart", "plugins", "installation"]
+---
+
+Before exploring ZKsync development with Hardhat, ensure your environment is properly set up. Follow these steps to install the necessary tools and
+meet the prerequisites for a smooth development experience.
+
+### Prerequisites
+
+**System requirements:**
+
+- Ensure your machine meets the [system requirements](%%zk_git_repo_era-compiler-solidity%%/tree/main#system-requirements) for running ZKsync and
+Hardhat. For Windows users, using Windows Subsystem for Linux (WSL 2) is highly recommended for better compatibility.
+
+**Node.js and package manager:**
+
+- [Node.js](https://nodejs.org) version 18 or newer.
+- Choose a package manager: [Yarn](https://classic.yarnpkg.com/en/docs/install) or [npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+
+**Familiarity with Ethereum development:**
+
+- Basic knowledge of deploying smart contracts on Ethereum is beneficial. Refer to
+   the [Contract Deployment](/build/developer-reference/ethereum-differences/contract-deployment) documentation if you need a refresher.
+
+**Wallet and testnet funds:**
+
+- You’ll need a wallet with sufficient ETH to deploy contracts.
+  See the [Network Faucets](/ecosystem/network-faucets) page to find a list of resources to obtain testnet funds.
+- Ensure you know how to [export your private key from MetaMask](https://support.metamask.io/hc/en-us/articles/360015289632-How-to-export-an-account-s-private-key).
+
+::callout{icon="i-heroicons-light-bulb"}
+If you're migrating an existing hardhat project to `zksync-era`, please refer to
+the [migration guide](/build/tooling/hardhat/guides/migrating-to-zksync) for
+detailed instructions.
+::
+
+### Installation steps
+
+**Install ZKsync CLI:**
+
+- The ZKsync CLI simplifies project setup and local testing. Install it globally with npm or yarn:
+
+::code-group
+
+  ```bash [yarn]
+    yarn global add zksync-cli
+  ```
+
+  ```bash [npm]
+    npm install -g zksync-cli
+  ```
+
+::
+
+**Create a new project:**
+
+- Use the ZKsync CLI to create a new Hardhat project. Choose the template based on your contract language (Solidity or Vyper):
+
+#### Solidity project
+
+```bash
+npx zksync-cli create demo --template hardhat_solidity
+```
+
+#### Vyper project
+
+```sh
+npx zksync-cli create demo --template hardhat_vyper
+```
+
+::callout{icon="i-heroicons-light-bulb"}
+Replace `<project-name>` with your desired project name. This command creates a `demo` folder and clones a Hardhat template project inside it.
+The downloaded project is already configured and contains all the required plugins.
+::
```

Comment:
Where is this `<project-name>`? Instead of making this a callout, move this description up at the beginning of this section. This is too late to describe what happened with the commands above.